app/build.gradle

```diff
@@ -103,7 +109,7 @@ dependencies {
     implementation "com.squareup.okhttp3:logging-interceptor:${rootConfiguration.okHttpVersion}"
 
     //For Link Previews
-    implementation "org.jsoup:jsoup:${rootConfiguration.jsoupVersion}" // required
```

Comment:
Why removed?

Reply:
I removed that by mistake but I added that.